packages/config/src/memory/memory.config.ts

```diff
@@ -115,9 +116,34 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
       } else if (isConfigImagery(obj)) {
         // TODO should this really overwrite existing tilesets
         this.put(ConfigProviderMemory.imageryToTileSet(obj));
-        this.imageryToTileSetByName(obj);
+        const tileSet = this.imageryToTileSetByName(obj);
+        allLayers.push(tileSet.layers[0]);
       }
     }
+    // Create an all tileset contains all raster layers
+    if (allLayers.length) this.createVirtualAllTileSet(allLayers);
+  }
+
+  createVirtualAllTileSet(layers: ConfigLayer[]): void {
+    const layerByName = new Map<string, ConfigLayer>();
+    // Set all layers as minZoom:32
+    for (const l of layers) {
+      const newLayer = { ...l, maxZoom: undefined, minZoom: 32 };
+      // TODO: This might overwrite the layer id for duplicated configImagery
+      if (layerByName.has(newLayer.name))
```

Comment:
It would be good to compare timestamps to see which imagery set is newer when overwriting it.

Reply:
also use `{` anytime you need a if over two lines.